xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -413,7 +421,12 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
 ##
 ##
 #if(!$replied)
-  #displayCommentForm()
+  #if($xwiki.hasAccessLevel('comment'))
+    #displayCommentButton()
+    <div id="CommentFormPlaceholder" style="display: none;">
```

Comment:
It is against the best practices to use inline style. You can hide this element from css, there is a `comments.css` file you can use.
I can see that there was some usage of inline style, just few lines above, but that doesn't mean it is a good example.
Same remark for the id name, to use `camelCase`.

Reply:
> It is against the best practices to use inline style.

Good catch. For info @manuelleduc AFAIR we have our WebStandards tests (in flavor-test-webstandards) that normally fails on those kind of things.

Reply:
Good to know. I wasn't aware of this test suite, thanks.